src/main/java/summarizer/team/NormalTeamEquipmentSummarizer.java

```diff
@@ -0,0 +1,37 @@
+package summarizer.team;
+
+import equipment.IEquipmentType;
+import team.ITeamMember;
+import equipment.AbstractEquipment;
+import equipment.uniform.UniformEquipmentType;
+import summarizer.SummarizerUtils;
+
+import java.util.List;
+import java.util.Map;
+
+public class NormalTeamEquipmentSummarizer implements ITeamEquipmentSummarizer {
+
+    @Override
+    public TeamSummarizerType getSummaryType() {
+        return TeamSummarizerType.NormalTeamEquipmentSummarizer;
+    }
+
+    @Override
+    public String getTeamSummary(Map<ITeamMember, List<AbstractEquipment>> teamEquipments) {
+        StringBuilder teamEquipmentSummarizer = new StringBuilder();
+        SummarizerUtils summarizerUtils = new SummarizerUtils();
+        for (ITeamMember teamMember: teamEquipments.keySet()) {
+            teamEquipmentSummarizer.append(teamMember.getName()).append(" from team ").append(teamMember.getTeamName())
```

Comment:
add newlines in the code between these long `append` chains for added readability